### PR TITLE
Add D4T into codeart

### PR DESCRIPTION
**Pull request by Narges-abbasii**

### CODE REVIEW

**Comment by m-zakeri** [comment]
Add only refactoring and testability measurement modules
not all files, not common files.

**Comment by Narges-abbasii** [comment]
I reviewed the files and made the necessary changes. Also, I put the Gorgeous/Resources directory into gitignore if it is not enough let me know thank you.

**Review by m-zakeri — CHANGES_REQUESTED** [review]
Many irrelevant files and changes. 
Please consider fixing the request.

**Comment by Narges-abbasii** [comment]
I resolved irrelevant files problem

File: requirements.txt
Comment (m-zakeri) [inline requirements.txt:1]: Requirements contain repeated items. 
Please check and fix.

**Review by m-zakeri — CHANGES_REQUESTED** [review]
Requirements contain repeated items. 
Generate the requirements file automatically to resolve problems

**Comment by Narges-abbasii** [comment]
removed duplicate versions of libraries in requirement.txt

**Review by m-zakeri — APPROVED** [review]

**Pull request merged**